modules/programs/rofi.nix

```diff
@@ -331,20 +335,27 @@ in {
       default = null;
       type = with types; nullOr (oneOf [ str path themeType ]);
       example = literalExample ''
-        with config.lib.formats.rasi; {
+        let
+          # config.lib.formats.rasi.mkLiteral unquotes a string, i.e.:
+          # l "abc" -> abc
+          l = config.lib.formats.rasi.mkLiteral;
```

Comment:
I'd prefer `inherit (config.lib.formats.rasi) mkLiteral;` instead of using a cryptic nameless function.

Reply:
Done.